spec/features/edit_page_spec.rb

```diff
@@ -9,6 +9,14 @@
     expect(page).to have_header("Edit #{customer.name}")
   end
 
+  it "has button to view customer details" do
+    customer = create(:customer)
+
+    visit edit_admin_customer_path(customer)
+
+    expect(page).to have_link("Show #{displayed(customer)}")
```

Comment:
Meant to push to this branch, but accidentally pushed to master. :confused:

This commit fixes up the include logic for the `displayed` helper.
https://github.com/thoughtbot/administrate/commit/9787489bef889e50cf0fc7f20c927b6481278a16